src/pages/Integrations/Create/CustomComponents/InlineAlert.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+
+const InlineAlert = ({
+  variant,
+  title,
+}: {
+  variant: AlertVariant;
+  title: string;
+}) => {
+  return (
+    <React.Fragment>
+      <Alert variant={variant} isInline isPlain title={title} />
+    </React.Fragment>
+  );
+};
```

Comment:
No need to wrap in function body and `React.Fragment`
```suggestion
}) => <Alert variant={variant} isInline isPlain title={title} />;
```

Reply:
Fixed